tsconfig.cjs.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+      "outDir": "dist/cjs",
+      "module": "CommonJS",
+      "target": "es2020",
+      "lib": ["es2020"]
```

Comment:
I think these two lines aren't needed, as they're just repeating the default in the extended config right?

Reply:
Yeah those can be removed

Reply:
done